app/mailers/referee_mailer.rb

```diff
@@ -5,8 +5,9 @@ def reference_request_email(application_form, reference)
     @candidate_name = application_form.full_name
 
     if FeatureFlag.active?('reference_form')
-      @reference_link = referee_interface_reference_feedback_url(token: reference.update_token!)
-      @decline_reference_link = 'link_to_decline_the_reference' #TODO: implement the flow for Referee decline to give references
+      unhashed_token = reference.update_token!
+      @reference_link = referee_interface_reference_feedback_url(token: unhashed_token)
+      @refuse_to_give_feedback_link = referee_interface_refuse_to_give_feedback_url(token: unhashed_token)
```

Comment:
Perhaps take the opportunity to move the calls to the URL helpers into the template, and only pass in the `@unhashed_token`.

Reply:
Let's do this in another PR.